enricher/fabric8/src/main/resources/META-INF/fabric8/enricher-default

```diff
@@ -5,3 +5,4 @@ io.fabric8.maven.enricher.fabric8.IconEnricher
 io.fabric8.maven.enricher.fabric8.ExposeEnricher,1000
 io.fabric8.maven.enricher.fabric8.SpringBootHealthCheckEnricher
 io.fabric8.maven.enricher.fabric8.KarafHealthCheckEnricher
+io.fabric8.maven.enricher.fabric8.PrometheusEnricher
```

Comment:
See the current `default-enrichers` from master how it is solved for the moment.